packages/scheduler/src/forks/SchedulerDOM.js

```diff
@@ -41,7 +41,7 @@ import {enableIsInputPending} from '../SchedulerFeatureFlags';
 
 let getCurrentTime;
 const hasPerformanceNow =
-  typeof performance === 'object' && typeof performance.now === 'function';
+  window.performance && typeof performance === 'object' && typeof performance.now === 'function';
```

Comment:
Question: Where is performance coming from in the latter checks?  If performance is on window should it not be:
```
window.performance && typeof window.performance === 'object' && typeof window.performance.now === 'function';
```

or 

```
const performance = window.performance || undefined;
const hasPerformanceNow = typeof performance === 'object' && typeof performance.now === 'function';
```

Reply:
Yeah, I find the reason. In my case, performance was set null some where, not undefined. it's weird.